Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -77,6 +78,7 @@ public class CarPlayManager: NSObject {
 
     private weak var navigationService: NavigationService?
     private var idleTimerCancellable: IdleTimerManager.Cancellable?
+    private let logger: OSLog = .init(subsystem: "com.mapbox.navigation", category: "CarPlayManager")
```

Comment:
Should we add a factory for loggers? So that we don't duplicate `subsystem` value everywhere.

Reply:
Or we can go with a simpler way: creating an internal constant for the log subsystem.

Reply:
We could potentially use `OSLogManager` but would probably want to consider using this manager for all of the SDK.